fidesctl/src/fidesapi/routes/crud.py

```diff
@@ -0,0 +1,90 @@
+# type: ignore
```

Comment:
Why is this needed to fix test failures?

Reply:
mypy is unhappy about some of the type magic going on here, this was already being ignored but at the config level. I moved it here for transparency